var/spack/repos/builtin/packages/flibcpp/package.py

```diff
@@ -14,7 +14,7 @@ class Flibcpp(CMakePackage):
     git = "https://github.com/swig-fortran/flibcpp.git"
     url = "https://github.com/swig-fortran/flibcpp/archive/v0.3.1.tar.gz"
 
-    version('master', branch='master')
+    version('0.5.2', sha256='b9b4eb6431d5b56a54c37f658df7455eafd3d204a5534903b127e0c8a1c9b827')
```

Comment:
Any reason to ignore `master`?

Reply:
Basically, now that `flibcpp` is stable I don't think exposing a "bleeding-edge" version is necessary.